scripts/README.md

```diff
@@ -109,15 +108,12 @@ set-window-option -g mouse on
 set -g default-terminal "tmux-256color"
 ```
 
-1.) create a `tmux` session
 
-`tmux new-session -s Demo`
```

Comment:
There is no longer a need to create a tmux session manually, see: https://github.com/input-output-hk/cardano-node/commit/d94c3194261d79dba931f1ac9872369032eabb0e